src/vm/docs/mod.rs

```diff
@@ -559,6 +559,16 @@ principal and executes `expr` with that context. It returns the resulting value
     example: "(as-contract (print tx-sender)) ;; Returns 'CTcontract.name"
 };
 
+const ASSERTS_API: SpecialAPI = SpecialAPI {
+    input_type: "(optional A) | (response A B), C",
```

Comment:
input_type should be: `bool, C`

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1127/commits/a921acfa2f20dcbab7e2b85adce3d1620bf0368e